.github/workflows/ci.yml

```diff
@@ -65,38 +65,29 @@ jobs:
     if: (github.event_name == 'pull_request') || (github.event_name == 'push' && github.ref == 'refs/heads/main')
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Setup Node.js
+      - name: Install Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "16"
-
+          node-version: "18"
       - name: Install Prettier
         run: npm install --global prettier
-
-      - name: Check Formatting
-        run: prettier --check "**/*.{graphql,yml,json,md}"
-
-      - name: Install Nightly Toolchain
+      - name: Install Stable Toolchain
         uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly
+          override: true
```

Comment:
It wasn't set before ... why do we need to set it to true now?

Reply:
I can change it back. I copy-pasted it from the recently merged changes today: https://github.com/tailcallhq/tailcall/blob/faa72e9ca7d40862ed7b34f45f9fee4650d53713/.github/workflows/bot.yml#L30-L32